projects/mongo-assessment.md

```diff
@@ -24,9 +24,8 @@ In this project, you will have 30 minutes to create a new database and a few doc
 3. Email your required screenshots to be reviewed at the end of the assessment.
 
 ## Setup Instructions
-
-- [ ] Know how to take screenshots on your machine. For mac, you can press `command + shift + 4` to draw a box around the part of the screen you want to capture (in this case, your command line terminal), and it should appear on your desktop with the name "Screenshot..." followed by a hash.
-- [ ] You should already have MongoDB installed after [this lesson](/Users/alodahl/Projects/curriculum/databases/mongo-db.md).
+- [ ] Know how to take screenshots on your machine.  For mac, you can press `command + shift + 4` to draw a box around the part of the screen you want to capture (in this case, your command line terminal), and it should appear on your desktop with the name "Screenshot..." followed by a hash.
+- [ ] You should already have MongoDB installed after [this lesson](/Users/alodahl/Projects/curriculum/electives/databases/mongo-db.md).
```

Comment:
this is clearly my fault, but could you change this from an alodahl URL to the real github elective URL?